src/Escaper.php

```diff
@@ -156,9 +157,20 @@ public function __construct(?string $encoding = null)
         $this->htmlSpecialCharsFlags = ENT_QUOTES | ENT_SUBSTITUTE;
 
         // set matcher callbacks
-        $this->htmlAttrMatcher = [$this, 'htmlAttrMatcher'];
-        $this->jsMatcher       = [$this, 'jsMatcher'];
-        $this->cssMatcher      = [$this, 'cssMatcher'];
+
```

Comment:
Changing this from an array callable was mainly about Psalm. Let me know if this should be reverted… I'll re-add to the baseline if so, but I don't know how to tell psalm that `[$this, 'somemethod']` is a `callable(list<string): string` ??

Reply:
IMO worth going back - adding another closure is quite wasteful.

What is psalm reporting when giving it an array? `callable(list<string>): string` should be the correct type, even in `array` form? :thinking: